### PR TITLE
Create SentimentAnalyzer.yml

### PR DESCRIPTION
**Pull request by arafattehsin**
It's inside _data/projects folder now.

### CODE REVIEW

**Comment by arafattehsin** [comment]
@ritwik12 I think you guys may want to have a look at the tags validation. Your bot always fail that. I ignored it now, please let me know if it works or not.

**Comment by ritwik12** [comment]
> @ritwik12 I think you guys may want to have a look at the tags validation. Your bot always fail that. I ignored it now, please let me know if it works or not.
@arafattehsin We have kept it case-sensitive with small case only so that there is no duplication of tags. Please try `sentimentanalysis` or `sentiment-analysis` and so on.

**Comment by shiftkey** [comment]
> I think you guys may want to have a look at the tags validation. Your bot always fail that. I ignored it now, please let me know if it works or not.

@arafattehsin please don't ignore these errors - they're in place to ensure tags are named in a sensible way (we originally had a mix of upper-case and lower-case labels, and this bot is trying to keep these formatted consistently across all the projects we list here)

File: _data/projects/SentimentAnalyzer.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: SentimentAnalyzer
+desc: Semantic Analyzer is an on-device (offline) open-source library to find out what customers think of your brand or topic by analyzing raw text for clues about positive or negative sentiment.
+site: https://github.com/arafattehsin/SentimentAnalyzer
+tags:
+  - SentimentAnalysis
```
Comment (shiftkey) [inline _data/projects/SentimentAnalyzer.yml]:
```suggestion
  - sentiment-analysis
```

File: _data/projects/SentimentAnalyzer.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  - TextAnalytics
```
Comment (shiftkey) [inline _data/projects/SentimentAnalyzer.yml]:
```suggestion
  - text-analytics
```

**Review by shiftkey — COMMENTED** [review]
These are what the bot is trying to warn us about before merging

**Comment by arafattehsin** [comment]
@shiftkey and @ritwik12 all good folks. I have changed as per the suggestions. Please let me know if you think this needs any change.

**Comment by ritwik12** [comment]
@arafattehsin Thanks!!

**Pull request merged**